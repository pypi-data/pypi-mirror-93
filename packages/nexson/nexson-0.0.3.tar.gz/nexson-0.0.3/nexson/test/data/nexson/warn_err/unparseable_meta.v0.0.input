{
"nexml": {
"@about": "#bogusStudyID", 
"@generator": "org.opentreeoflife.api.nexsonvalidator.nexson_nexml", 
"@id": "bogusStudyID", 
"@nexml2json": "0.0.0", 
"@version": "0.9", 
"@xmlns": {
"$": "http://www.nexml.org/2009", 
"nex": "http://www.nexml.org/2009", 
"ot": "http://purl.org/opentree/nexson", 
"skos": "http://www.w3.org/2004/02/skos/core#", 
"tb": "http://purl.org/phylo/treebase/2.0/terms#", 
"xsd": "http://www.w3.org/2001/XMLSchema#", 
"xsi": "http://www.w3.org/2001/XMLSchema-instance"
}, 
"meta": [
{
"$": "jane q. public", 
"@datatype": "xsd:string", 
"@xsi:type": "nex:LiteralMeta"
}, 
"http://purl.org/phylo/treebase/phylows/study/TB2:S1623", 
5,
4.5,
true, 
{
"$": "jane q. public", 
"@datatype": "xsd:string", 
"@property": "ot:curatorName", 
"@xsi:type": "nex:LiteralMeta"
}, 
{
"@href": "http://purl.org/phylo/treebase/phylows/study/TB2:S1623", 
"@rel": "ot:dataDeposit", 
"@xsi:type": "nex:ResourceMeta"
}, 
{
"$": "a", 
"@datatype": "xsd:string", 
"@property": "ot:focalClade", 
"@xsi:type": "nex:LiteralMeta"
}, 
{
"$": "b", 
"@datatype": "xsd:string", 
"@property": "ot:studyId", 
"@xsi:type": "nex:LiteralMeta"
}, 
{
"@href": "http://dx.doi.org/10.1073/pnas.0709121104", 
"@datatype": "xsd:string", 
"@property": "ot:studyPublication", 
"@xsi:type": "nex:ResourceMeta"
}, 
{
"$": "Some free, text here.", 
"@datatype": "xsd:string", 
"@property": "ot:studyPublicationReference", 
"@xsi:type": "nex:LiteralMeta"
}, 
{
"$": 2014, 
"@datatype": "xsd:int", 
"@property": "ot:studyYear", 
"@xsi:type": "nex:LiteralMeta"
}
], 
"otus": [], 
"trees": []
}
}
