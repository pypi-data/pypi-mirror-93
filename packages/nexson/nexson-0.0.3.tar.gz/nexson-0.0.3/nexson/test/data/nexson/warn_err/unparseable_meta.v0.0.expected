{
  "errors": {
    "INCORRECT_VALUE_TYPE": {
      "data": [
        {
          "expected": "integer", 
          "key": "ot:focalClade", 
          "type": "unicode"
        }
      ], 
      "refersTo": {
        "@idref": "bogusStudyID", 
        "@top": "nexml"
      }
    }, 
    "UNPARSEABLE_META": {
      "data": [
        {
          "$": "jane q. public", 
          "@datatype": "xsd:string", 
          "@xsi:type": "nex:LiteralMeta"
        }, 
        "http://purl.org/phylo/treebase/phylows/study/TB2:S1623", 
        5, 
        4.5, 
        true
      ], 
      "refersTo": {
        "@idref": "bogusStudyID", 
        "@top": "nexml"
      }
    }
  }, 
  "warnings": {}
}
